### ISSUE TITLE
No longer download sunday on Saturday night

### ISSUE THREAD

**Issue by jmorgie** [kebernet/shortyz#20]
Used to be able to download next day's puzzles on prior evening. Now this no longer works. Can you restore functionality?

**Comment by pan2** [comment]
Agreed. In general it seems the app disallows downloading any puzzle ahead of the current date. On the west coast, the New York Times crossword is available before midnight but not via the app.

**Comment by jmorgie** [comment]
Also the Wall street journal is now downloading broken … have they changed their format ??

 

jm
